### ISSUE TITLE
Tutorial: Simple LSTM

### ISSUE THREAD

**Issue by iConBarca** [pytorch/fairseq#1427]
![image](https://user-images.githubusercontent.com/23134066/69554510-de98d080-0fdc-11ea-9d83-85578bbad183.png)
Does anyone know how to solve it?
Thank u.

**Comment by myleott** [comment]
fix incoming

**Comment by iConBarca** [comment]
> fix incoming

python train.py  data-bin/iwslt14.tokenized.de-en --arch tutorial_simple_lstm --encoder-dropout 0.2 --decoder-dropout 0.2 --optimizer adam --lr 0.005 --lr-shrink 0.5 --max-tokens 12000
Fix this ?

**Comment by myleott** [comment]
Sorry, I meant to say that I'm working on a fix and will submit it shortly

**Comment by myleott** [comment]
Fixed here: https://github.com/pytorch/fairseq/commit/5d9392df979553827082afee836c1c386fc7cbda
